data/conf/iceberg/llap/hive-site.xml

```diff
@@ -397,4 +397,39 @@
         <name>hive.lock.sleep.between.retries</name>
         <value>2</value>
     </property>
+    
+    <!-- -- Iceberg REST catalog specific properties -->
+    <property>
+        <name>iceberg.catalog.type</name>
+        <value>rest</value>
+    </property>
+    <property>
+        <name>iceberg.rest-catalog.uri</name>
```

Comment:
`iceberg.rest-catalog.prefix` is missing
ref https://trino.io/docs/current/object-storage/metastores.html#iceberg-rest-catalog

Reply:
iceberg supports prefix, see spec
https://github.com/apache/iceberg/blob/main/open-api/rest-catalog-open-api.yaml#L42

Reply:
ok